benchmarks/models.py

```diff
@@ -49,6 +51,7 @@ def _generate_square_subsequent_mask(self, sz):
         return mask
 
     def forward(self, src):
+        # TODO(anj-s): Fix the data format so that we have [seq_len, batch_size, embedding dim].
```

Comment:
it is unclear from this comment what's the current format of src. did you run it and inspect the shape of src?

Reply:
Added an extra comment to clarify. Yes, I did run it to confirm that we have batch size in the first dimension for synthetic data.